src/com/tkjngine/gametable/monsterwounds/MonsterData.java

```diff
@@ -0,0 +1,69 @@
+package com.tkjngine.gametable.monsterwounds;
+
+import java.util.*;
+
+public class MonsterData extends Observable
+{
+    private int maxWounds = 0;
+    private int wounds = 0;
+
+    public int getMaxWounds()
+    {
+        return maxWounds;
+    }
+
+    public void setMaxWounds(int maxWounds)
+    {
+        if (this.maxWounds == maxWounds
+            && this.wounds <= maxWounds)
+        {
+            return;
+        }
+
+        this.maxWounds = maxWounds;
+        if (this.wounds > maxWounds) {
+            this.wounds = maxWounds;
+        }
+        notifyChange();
+    }
+
+    public int getWounds()
+    {
+        return wounds;
+    }
+
+    public void setWounds(int wounds)
+    {
+        if (this.wounds != wounds
+            && wounds >= 0
+            && wounds <= this.maxWounds
+        ) {
+            this.wounds = wounds;
+            notifyChange();
+        }
+    }
+
+    public void incWounds()
+    {
+        this.setWounds(wounds + 1);
+    }
+
+    public void decWounds()
+    {
+        this.setWounds(wounds - 1);
+    }
+
+    private void notifyChange()
+    {
+        setChanged();
+        notifyObservers();
+    }
+
+    /**
+     * Re-initialises data - e.g. when about to load from a save
+     */
+    public void resetData()
```

Comment:
I get a warning that this method is not used, not sure if something is going to call it in the future?